### PR TITLE
updates and additions

### PR DESCRIPTION
**Pull request by CWells1**

### CODE REVIEW

File: .gitignore
Comment (tkluysk) [inline .gitignore:1]: Just gitignore the 'book' folder, instead of every file in it.

Comment (tkluysk) [reply .gitignore:1]: Done

File: CreatorNodeDocumentation/src/SUMMARY.md [hunk 1]
```diff
@@ -41,194 +41,194 @@
...
+  - [Join strings](nodes/StringJoin/documentation.md)
+  - [Split string](nodes/StringSplit/documentation.md)
+  - [String switch](nodes/StringSwitch/documentation.md)
+  - [JSON builder](nodes/JsonBuilder/documentation.md)
 - [Utility](nodeSections/utilitySection.md)
-  - [JSON builder]()
-  - [Combine data]()
+  - [JSON builder](nodes/JsonBuilder/documentation.md)
+  - [Combine data](nodes/CombineData/documentation.md)
   - [Ephemeral state]()
```
Comment (tkluysk) [inline CreatorNodeDocumentation/src/SUMMARY.md:230]: omission or not done yet?

File: CreatorNodeDocumentation/src/concepts/GeneralConcepts/ephemeralState.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Ephemeral State
 
-Graphs have a boolean value representing their ephemeral state, and this state is exposed to graph logic through the **ephemeral** node.
+Graphs have a boolean value representing their ephemeral state, and this state is exposed to graph logic through the [**Ephemeral**](/nodes/Ephemeral/documentation.md). A graph is in an ephemeral state when parameters are actively being configured/adjusted. For example, when one is dragging a number slider in the Parameter panel.
```
Comment (tkluysk) [inline CreatorNodeDocumentation/src/concepts/GeneralConcepts/ephemeralState.md:3]: a graph *can be* in a ephemeral state...

it's driven programmatically via the SDK (including the CPV), so applications using the Materia service can put Materia in an ephemeral state ay time,

File: CreatorNodeDocumentation/src/introduction.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ Trimble Creator's core concept is a graph. Trimble Creator graphs represent logi
 
 Graphs are a collection of connected nodes. Individual nodes represent compute functions, like the creation of a polygonal box, the noise math operation, or the act of joining two curves.
 
+This graph <a href="https://creator.trimble.com/graph?assetURI=whp:f790227f-a092-4595-86ca-f6e7d2eeac14&version=latest" target="_blank">HERE</a> has every node that exists within Trimble Creator, open it up to see them all!
```
Comment (tkluysk) [inline CreatorNodeDocumentation/src/introduction.md:11]: dangerous to say 'every node', as this now needs to be maintained rigorously to make this true

File: CreatorNodeDocumentation/src/introduction.md [hunk 4]
```diff
@@ -8,6 +8,8 @@ Trimble Creator's core concept is a graph. Trimble Creator graphs represent logi
 
 Graphs are a collection of connected nodes. Individual nodes represent compute functions, like the creation of a polygonal box, the noise math operation, or the act of joining two curves.
 
+This graph <a href="https://creator.trimble.com/graph?assetURI=whp:f790227f-a092-4595-86ca-f6e7d2eeac14&version=latest" target="_blank">HERE</a> has every node that exists within Trimble Creator, open it up to see them all!
+
 _NOTE: This documentation is currently in an 'in-progress' state and is therefore incomplete. Updates will be made periodically._
```
Comment (tkluysk) [inline CreatorNodeDocumentation/src/introduction.md:13]: Should not be a statement in documentation.

Just say that the demo graph has most of the nodes, and make sure we have a process for updating the graph now and then.

File: CreatorNodeDocumentation/src/SUMMARY.md (hunk 1 above)
Comment (CWells1) [reply CreatorNodeDocumentation/src/SUMMARY.md:230]: missed, adding in now.

**Pull request merged**